src/highlight_manager.ts

```diff
@@ -135,8 +155,8 @@ export class HighlightManager implements Disposable {
         }
     }
 
-    private applyHLGridUpdates(pendingUpdates: PendingUpdates<number>): void {
-        for (const [grid, update] of pendingUpdates.entries()) {
+    private applyHLGridUpdates(): void {
+        for (const [grid, update] of this.pendingGridUpdates.entries()) {
```

Comment:
`update` is a bit clearer in this case, IMO, because there is an actual mutation performed (the `update` function calls `this.highlightProvider.processHLCellsEvent`. That said, if you feel strongly I'll change it